.github/workflows/checks.yml

```diff
@@ -42,7 +48,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: volta-cli/action@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: "package.json"
```

Comment:
I don't think we need this, setup node should use the volta version automatically.